vm/src/value.rs

```diff
@@ -439,7 +439,23 @@ impl<'a, 't> InternalPrinter<'a, 't> {
             Value::Userdata(ref data) => arena.text(format!("{:?}", data)),
             Value::Thread(thread) => arena.text(format!("{:?}", thread)),
             Value::Byte(b) => arena.text(format!("{}", b)),
-            Value::Int(i) => arena.text(format!("{}", i)),
+            Value::Int(i) => {
+                use base::types::BuiltinType;
+                match **self.typ {
+                    Type::Builtin(BuiltinType::Int) => arena.text(format!("{}", i)),
+                    Type::Builtin(BuiltinType::Char) =>
+                        if i < ::std::u32::MAX as isize {
+                            match ::std::char::from_u32(i as u32) {
+                                Some('"') => arena.text(format!("'{}'", '"')),
+                                Some(c) => arena.text(format!("'{}'", c.escape_default())),
+                                None => unreachable!(),
```

Comment:
Why the special case for `"`?

Also kind of concerning that we are storing unicode chars as `isize`s. What if they are larger than 32 bits? Not really an issue for this PR though 🤔

Reply:
`char::escape_default` returns `'\"'` for input `'"'`.
https://doc.rust-lang.org/std/primitive.char.html#method.escape_default
I thought that input `'"'` to repl should corresponds to output `'"',` and treated it specially.